plugin/src/main/kotlin/com/facebook/testing/screenshot/build/ScreenshotsPlugin.kt

```diff
@@ -71,11 +71,19 @@ class ScreenshotsPlugin : Plugin<Project> {
   private fun generateTasksFor(project: Project, variant: TestVariant) {
     variant.outputs.all {
       if (it is ApkVariantOutput) {
+        val cleanScreenshots =createTask(
+            project,
+            CleanScreenshotsTask.taskName(variant),
+            variant,
+            CleanScreenshotsTask::class.java)
+        project.tasks.getByName("clean")
```

Comment:
`getByName` could potentially throw an `UnknownTaskException` , which we should probably handle gracefully.

Also, doesn't the default clean task delete the entirety of the `build/` directory? Do we need to have it depend on each variant's cleanScreenshots task?